test/branches/branches.test.js

```diff
@@ -11,8 +11,21 @@ const merge = (branches, source, target, tag) => {
     getBranch(branches, target).tags
   );
 };
+const remoteBranches = [];
+const repositoryUrl = 'repositoryUrl';
+let expand, getTags, getBranches;
 
-test('Enforce ranges with branching release workflow', async (t) => {
+test.beforeEach(async (t) => {
+  getTags = (await td.replaceEsm('../../lib/branches/get-tags.js')).default;
+  expand = (await td.replaceEsm('../../lib/branches/expand.js')).default;
+  getBranches = (await import('../../lib/branches/index.js')).default;
+})
+
+test.afterEach.always((t) => {
+  td.reset();
+});
+
+test.serial('Enforce ranges with branching release workflow', async (t) => {
```

Comment:
i dont like making these tests run serially, but it was necessary to avoid conflicts for stub setup between tests in this file when the tests execute in parallel. i think we could rework the stub setup between tests to avoid that situation, but i think that should be given attention separately from the ESM effort